programs/mango-v4/tests/program_test/mango_client.rs

```diff
@@ -1765,6 +1808,49 @@ impl ClientInstruction for AccountCloseInstruction {
     }
 }
 
+pub struct AccountSettleFeesWithMngo {
+    pub group: Pubkey,
+    pub owner: TestKeypair,
+    pub account: Pubkey,
+    pub dao_account: Pubkey,
```

Comment:
no need to pass the group or dao_account explicitly, can get the group key from the account, load it and fetch the dao_account